assets/sass/vendor/_mdc-checkbox.scss

```diff
@@ -31,6 +82,42 @@
 			vertical-align: middle;
 			width: 100%;
 			z-index: 1;
+
+			&:focus {
+				~ .mdc-checkbox__background::before {
+					opacity: 1;
+				}
+			}
+
+			&:enabled:not(:checked, :indeterminate),
+			&:checked:enabled {
+				~ .mdc-checkbox__background {
+					border-color: $c-black;
+				}
+			}
+
+			&:enabled:not(:checked, :indeterminate) {
+				~ .mdc-checkbox__background {
+					background-color: transparent;
```

Comment:
Is this background colour style definitely needed? Again, maybe I'm missing something, but it seems OK when I remove it...

Reply:
Thanks for clarifying, sorry I totally missed what now seems quite obvious!